browser/ui/webui/brave_shields/shields_panel_data_handler.cc

```diff
@@ -157,6 +157,8 @@ void ShieldsPanelDataHandler::UpdateSiteBlockInfo() {
       active_shields_data_controller_->GetHttpRedirectsList();
   site_block_info_.is_shields_enabled =
       active_shields_data_controller_->GetBraveShieldsEnabled();
+  site_block_info_.is_shields_managed =
```

Comment:
I pinged @nullhook about inconsistent usage here `is_shields` vs `GetBraveShields`. `Is` is preferred so I think we should have `IsBraveShieldsEnabled` and `is_brave_shields_enabled` and `is_brave_shields_managed`

Reply:
I removed all